apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.theme.scss

```diff
@@ -1,8 +1,10 @@
 @use 'sass:map';
 @use '@angular/material' as mat;
 @use '../../../styles/constants/palettes' as palettes;
+@use '../../../styles/breakpoints' as breakpoints;
 
 $gray-palette: map.get(palettes.$palettes, 'gray');
+$blue-palette: map.get(palettes.$palettes, 'blue');
```

Comment:
This should come from the `$theme` variable. Use `mat.get-theme-color($theme, accent, lighter)` to get blue 500

Reply:
Remove this line